security/server/ipsec_vici.go

```diff
@@ -105,6 +105,74 @@ type rekey_connection struct {
 	Reauth     bool   `vici:"reauth"`
 }
 
+type list_sas struct {
+	Ike        string `vici:"ike"`
+	IkeId      string `vici:"ike"`
+	Child      string `vici:"child"`
+	ChildId    string `vici:"child"`
```

Comment:
Similarly, should this be `vici:"child-id"`?

Reply:
Another nice find!